account_invoice_validate_send_email/data/email_templates.xml

```diff
@@ -4,11 +4,11 @@
         <field name="name">請求書送付のご案内</field>
         <field
             name="email_from"
-        ><![CDATA["${object.company_id.name}" <${(object.company_id.email or user.email)|safe}>]]></field>
-        <field name="reply_to">${object.company_id.email}</field>
+        ><![CDATA["${object.invoice_mail_template_id.name}" <${(object.invoice_mail_template_id.email or user.email)|safe}>]]></field>
+        <field name="reply_to">${object.invoice_mail_template_id.email}</field>
```

Comment:
@tack-yabeQuartile The changes in this file look incorrect/unnecessary. Please revert these changes for now.